xml/art_minvm.xml

```diff
@@ -23,7 +23,7 @@
   </dm:docmanager>
   <abstract>
    <para>
-    The following document provides an overview of &sls; &jeos; and describes
+    The following document provides an overview of &sls; &minvm; and describes
```

Comment:
Would it help to add a references to the old name? Some customers may know the old name. Maybe something like "(formerly known as JeOS)"?

Reply:
Good point. I've added the reference.